.goreleaser.yaml

```diff
@@ -298,17 +305,19 @@ dockers:
 
 docker_manifests:
   # https://goreleaser.com/customization/docker_manifest/
-  - name_template: "{{ .Env.REGISTRY }}/rancher/fleet:{{ .Tag }}"
+  - name_template: "docker.io/rancher/fleet:{{ .Tag }}"
     id: fleet-manifest-public
+    skip_push: "{{ .Env.IS_HOTFIX }}"
```

Comment:
There is no [`skip_build` support for this type](https://goreleaser.com/customization/docker_manifest/#customization), so I don't know if it will still run with `dry-run`. If that's the case, it may fail due to the images not. being present locally? Should I remove `skip_build` for the images just in case?